lib/modules/manager/npm/post-update/rules.ts

```diff
@@ -19,8 +19,13 @@ export function processHostRules(): HostRulesResult {
   const npmHostRules = hostRules.findAll({
     hostType: 'npm',
   });
-  logger.debug(`Found ${npmHostRules.length} npm host rule(s)`);
-  for (const hostRule of npmHostRules) {
+  // Include host rules without specific type to mimic the behavior used when determining dependencies with updates.
+  const noTypeHostRules = hostRules
+    .getAll()
+    .filter((rule) => rule.hostType === null || rule.hostType === undefined);
+  const effectiveHostRules = npmHostRules.concat(noTypeHostRules);
+  logger.debug(`Found ${effectiveHostRules.length} npm host rule(s)`);
```

Comment:
Can you log how many were found with hostType=npm and then how many were found with empty hostType?

Reply:
This should be resolved already?